apps/backend/src/libs/modules/controller/libs/types/api-handler-options.type.ts

```diff
@@ -4,12 +4,14 @@ type APIHandlerOptions<
 	body: T["body"];
 	params: T["params"];
 	query: T["query"];
+	user: T["user"];
```

Comment:
It's not correct to add user in this type, because we will not always have a user. For white routes there will be no user. You can use type casting in auth controller instead